judge/views/contests.py

```diff
@@ -595,16 +612,27 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class CachedContestCalendar(ContestCalendar):
-    def render(self):
-        key = 'contest_cal:%d:%d' % (self.year, self.month)
-        cached = cache.get(key)
-        if cached is not None:
-            return HttpResponse(cached)
-        response = super(CachedContestCalendar, self).render()
-        response.render()
-        cached.set(key, response.content)
-        return response
+class ContestICal(TitleMixin, ContestListMixin, BaseListView):
+    def generate_ical(self):
+        cal = ICalendar()
+        cal.add('prodid', '-//DMOJ//NONSGML Contests Calendar//')
```

Comment:
Hm, mind taking a look at the spec? I think this should be changed to "MCPT" but I'm not *quite* certain.

Reply:
`prodid` is the name of the generator[^1]. It should be either "WLMOJ" or "DMOJ". I tihnk "DMOJ" is fine as a) the iCal code has (basically) no changes from upstream and b) less diffs between upstream.
[^1]: https://www.kanzaki.com/docs/ical/prodid.html

Reply:
Was the direction I was leaning as well. Let's keep `DMOJ`.